### PR TITLE
Fix bug allowing the creation of a note when the Trash folder is empty (#537)

### PR DESCRIPTION
**Pull request by rubenromani**
Issue https://github.com/nuttyartist/notes/issues/537:  Bug allowing the creation of a note when the Trash folder is empty

### CODE REVIEW

**Comment by nuttyartist** [comment]
Awesome, thanks for that! I'll wait for one more review and we can merge this.

**Review by zjeffer — CHANGES_REQUESTED** [review]
This fixes the bug, but causes the console to print this log with every letter you type in that (invalid) note:

`NoteEditorLogic::onTextEditTextChanged() : m_currentNote is not valid`

It would be better if the text editor was not editable at all, if the trash folder is selected and empty. Maybe find a way to "disable" the editor?

**Comment by nuttyartist** [comment]
Closing this due to #642. Thanks, @rubenromani!

**Pull request closed without merging**
